migration/migration.go

```diff
@@ -439,6 +439,16 @@ func GetMigrations() Migrations {
 
 	// Version 100
 	m = append(m, steps{ExecuteSQLFile("100-add-rules-to-wit.sql")})
+	
+	m = append(m, steps{
+		func(db *sql.Tx) error {
+			_, err := db.Exec("DROP TABLE IF EXISTS userspace_data")
```

Comment:
@michaelkleinhenz something seems to be off here because that clearly looks like a commit that @tinakurian made a few days ago ;)

Reply:
Somehow Tinas commits are always spilling into my branches. I don't know why.

Reply:
Ah, no, that may be due to a conflict I had with master. I solved it, but I may have added an additional extra blank line there. So this may be the reason this has ended up in here.

Reply:
And just to mention: this is a classic example where I'd have objected this commit. There is not a single comment line there that explains what this does. This is a complex piece there and this looks like a random code fragment was thrown in. Just for nitpicking.

Reply:
It is not any way more complex than the line above: `m = append(m, steps{ExecuteSQLFile("100-add-rules-to-wit.sql")})`. The migration code was designed to be able to be called from Go directly and Tina is doing that. So no problem here.

Reply:
I don't agree. The index of the migration is important. It is used in the test. This plus Preeti's PR that is also not using the correct approach is not causing massive confusion here.

Reply:
Plus, the test on the description PR is completely bonkers:

```
func testMigration101TypeGroupHasDescriptionField(t *testing.T) {
	migrateToVersion(t, sqlDB, migrations[:101], 101)
	require.False(t, dialect.HasColumn("work_item_type_groups", "description"))
}
```
(I changed the signature to have the numbering).

This one tests nothing. It imports the "old" version before the migration, then tests if the field is not there (it always is). This should be:

```
func testMigration101TypeGroupHasDescriptionField(t *testing.T) {
	migrateToVersion(t, sqlDB, migrations[:102], 102)
	require.True(t, dialect.HasColumn("work_item_type_groups", "description"))
}
```

This is the kind of stuff that really annoys me: we're nitpicking on details that are not relevant, but let broken code thru.